app/views/trainees/check_details/show.html.erb

```diff
@@ -42,16 +42,21 @@
       Personal details and education
     </h2>
 
-    <%= render Trainees::Confirmation::PersonalDetails::View.new(trainee: @trainee) %>
-    <%= render Trainees::Confirmation::ContactDetails::View.new(trainee: @trainee) %>
-    <%= render Trainees::Confirmation::Diversity::View.new(trainee: @trainee) %>
-    <%= render Trainees::Confirmation::Degrees::View.new(trainee: @trainee, show_add_another_degree_button: false, show_delete_button: true) %>
+    <%= render Trainees::Sections::View.new(trainee: @trainee, trn_submission: @trn_submission, section: :personal_details) %>
```

Comment:
I wonder whether passing the whole `@trn_submission` form object is a little overkill here, when we could pass in the error boolean and the section's state?

Reply:
Had a discussion offline and decided to leave this since we are planning to do another refactor